numpy/testing/utils.py

```diff
@@ -903,12 +908,12 @@ def compare(x, y):
         # casting of x later.
         dtype = result_type(y, 1.)
         y = array(y, dtype=dtype, copy=False, subok=True)
-        z = abs(x-y)
+        z = abs(x - y)
 
         if not issubdtype(z.dtype, number):
             z = z.astype(float_)  # handle object arrays
 
-        return around(z, decimal) <= 10.0**(-decimal)
+        return z < 1.5 * 10.0**(-decimal)
```

Comment:
I had that, but changed it on account of
```
nt.assert_almost_equal([1.5], [0], decimal=0)
```
failing in master due to rounding 1.5 up. Note that
```
nt.assert_almost_equal([1.49999], [0], decimal=0)
```
does pass (just to assure everyone the 1.5 is the proper factor).